sales.html

```diff
@@ -11,10 +11,13 @@
 	<body>
     <h1>Enter your Sales</h1>
 
-    <form id="salesInput">
+    <form id="salesForm">
       <fieldset>
         <legend>Pat's Cookies Sales Data Input</legend>
+        
 
+        <!-- What do you call these identifiers?? -->
+        
         <label for="where">Store Location: </label>
         <input name="where" type="text" size="15" />
```

Comment:
Just so you know you can use `type="number"` for the min, max, and average to ensure that the user enters a number value.